### PR TITLE
mips fixes

### PR DESCRIPTION
**Pull request by joewholden**
I'm working on getting lxd running on mips and this WIP PR fixes some low hanging fruit, there are some other issues like Rdev uint32/64 mismatches which may affect other platforms so they're not going to be included in this PR.

### CODE REVIEW

**Comment by brauner** [comment]
Hey, thanks for the patch. That's great. We only require that you sign-off your commits. Could you do that please? :)

**Comment by joewholden** [comment]
Yup I always forget! There are a few more things I need to fix before this is ready however

File: lxd/util/fs_64bit.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// +build amd64 ppc64 ppc64le arm64 s390x
+// +build amd64 ppc64 ppc64le arm64 s390x mips64 mips64le
```
Comment (brauner) [inline lxd/util/fs_64bit.go:1]: Maybe we could document arches that we as upstream support and others which are supported by the community? @stgraber wdyt?

Comment (stgraber) [reply lxd/util/fs_64bit.go:1]: That's kinda up to the distros as far as what they want to support. As upstream LXD, I'm fine accepting fixes for various distros.

As the packager for the snap, I know what architecture I build and test for.

It's a bit odd having the two hats but I don't think we need a stronger stance around that.
Right now as upstream, our CI is pretty much x86 only, for non-x86, it's effectively snap CI. Just happens to be run by the same people ;)

**Pull request merged**